### PR TITLE
Export fund requests, Default validators and Auto fund approval

### PR DESCRIPTION
**Pull request by dev-rminds**

### CODE REVIEW

File: src/sources/forus-webshop/pug/tpl/pages/fund-request-auto.pug [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                        li U bent 18 jaar of ouder
+                        li U bent geen student
+                        li U woont in Nijmegen
+                        li U voldoet aan de maximale inkomensgrens (netto) voor de Meedoen-regeling 2020:
+                    
+                    p.sign_up-pane-text 
+                        | Door te verklaren dat u aan de voorwaarden voldoet, 
+                        | gaat u er mee akkoord dat uw aanvraag gecontroleerd zal worden, en dat het bedrag van €150,- 
+                        | zal worden teruggevorderd indien u achteraf niet blijkt te voldoen.
+
```
Comment (maxvisser) [inline src/sources/forus-webshop/pug/tpl/pages/fund-request-auto.pug:127]: Add the table in figma here.

**Comment by maxvisser** [comment]
Added some translation. 

I can't find anything wrong with the branch with some minimal use case testing. I would propose to already add it to release 6.1 and test it more fully with different usecases. 

@dev-rminds do you still want to work on this more; If not please switch the branch from draft to pr

**Comment by maxvisser** [comment]
I miss the html table with hardcoded criteria from Figma on the 'meedoen_2020_eligible' checkbox step.

**Comment by maxvisser** [comment]
discussion 3 januari:
hide auto asign and auto validation on fund edit when digid is disabled

**Comment by jamalv** [comment]
## Update webshop homepage:
With DigiD enabled; text on the frontend should be updated:

- [ ] tekst: https://github.com/teamforus/forus/issues/1269#issuecomment-564510423

## Update validation step:
I cant currently test, but the text on the checkbox step needs to be:

- [x] Update validation step tekst

Screenshot 2020-01-13 at 17 42 51

```
Inkomensgrens voor de Mee-doen regeling 2020:  

                                                                  vanaf 18 jaar tot uw pensioenleeftijd          vanaf pensioenleeftijd                  indicatie Zorg met verblijf

Alleenstaand                                                                          1166.00                                         1305                                                          375

Alleenstaande ouder                                                             1496.00                                        1584                                                           505

Getrouwd/samenwonend                                                    1661.00                                        1760                                                           590

Wat wordt verstaan onder inkomen:

Wat hoort bij het inkomen?
Alle inkomsten uit werk, uitkering, pensioen, alimentatie, kamerhuur, kostgeld, heffingskorting belastingdienst. 
Wilt u weten op welke heffingskortingen u recht heeft? Kijk dan op de website van de Belastingdienst: www.belastingdienst.nl.

Wat hoort niet bij het inkomen?
Zorgtoeslag, kinderopvangtoeslag, huurtoeslag, en doeluitkeringen of vergoedingen (bijvoorbeeld schadevergoedingen).
```

**Comment by jamalv** [comment]
Figma (old): https://www.figma.com/file/buNhwrhiuG23hrxWZRGkrC/Nijmegen?node-id=33%3A87

**Comment by maxvisser** [comment]
image

**Comment by maxvisser** [comment]
localhost_5520_

to which page does 'Voorwaarden' need to link?

**Comment by jamalv** [comment]
https://www.nijmegen.nl/diensten/uitkering-schulden-laag-inkomen/meedoenregeling/

**Review by maxvisser — APPROVED** [review]

**Pull request merged**